### ISSUE TITLE
Stacktrace when trying to run tests in PyCharm 2.0 with JsTestDriverPlugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#296]
```
What steps will reproduce the problem?
1. Run all tests based on configuration file in directory

What is the expected output?
Tests pass

 What do you see instead?
Error during dispatching of 
java.awt.event.MouseEvent[MOUSE_RELEASED,(668,46),absolute(668,68),button=1,modi
fiers=Button1,clickCount=1] on frame1: 
com.intellij.execution.testframework.sm.SMTestRunnerConnectionUtil.attachRunner(
Ljava/lang/String;Lcom/intellij/execution/process/ProcessHandler;Lcom/intellij/e
xecution/testframework/TestConsoleProperties;Lcom/intellij/execution/configurati
ons/RunnerSettings;Lcom/intellij/execution/configurations/ConfigurationPerRunner
Settings;)Lcom/intellij/execution/testframework/ui/BaseTestsOutputConsoleView;
java.lang.NoSuchMethodError: 
com.intellij.execution.testframework.sm.SMTestRunnerConnectionUtil.attachRunner(
Ljava/lang/String;Lcom/intellij/execution/process/ProcessHandler;Lcom/intellij/e
xecution/testframework/TestConsoleProperties;Lcom/intellij/execution/configurati
ons/RunnerSettings;Lcom/intellij/execution/configurations/ConfigurationPerRunner
Settings;)Lcom/intellij/execution/testframework/ui/BaseTestsOutputConsoleView;
    at com.google.jstestdriver.idea.execution.TestRunnerState.execute(TestRunnerState.java:122)
    at com.intellij.execution.runners.DefaultProgramRunner.doExecute(DefaultProgramRunner.java:35)
    at com.intellij.execution.runners.GenericProgramRunner$1.execute(GenericProgramRunner.java:85)
    at com.intellij.execution.impl.ExecutionManagerImpl$2.run(ExecutionManagerImpl.java:156)
    at com.intellij.execution.impl.ExecutionManagerImpl.compileAndRun(ExecutionManagerImpl.java:133)
    at com.intellij.execution.impl.ExecutionManagerImpl.startRunProfile(ExecutionManagerImpl.java:186)
    at com.intellij.execution.runners.GenericProgramRunner.execute(GenericProgramRunner.java:78)
    at com.intellij.execution.runners.GenericProgramRunner.execute(GenericProgramRunner.java:62)
    at com.intellij.execution.ProgramRunnerUtil.executeConfiguration(ProgramRunnerUtil.java:80)
    at com.intellij.execution.ProgramRunnerUtil.executeConfiguration(ProgramRunnerUtil.java:89)
    at com.intellij.execution.ExecutorRegistryImpl$ExecutorAction.actionPerformed(ExecutorRegistryImpl.java:233)
    at com.intellij.openapi.actionSystem.impl.ActionButton.a(ActionButton.java:162)
    at com.intellij.openapi.actionSystem.impl.ActionButton.a(ActionButton.java:132)
    at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:303)
    at java.awt.Component.processEvent(Component.java:6138)
    at java.awt.Container.processEvent(Container.java:2085)
    at java.awt.Component.dispatchEventImpl(Component.java:4735)
    at java.awt.Container.dispatchEventImpl(Container.java:2143)
    at java.awt.Component.dispatchEvent(Component.java:4565)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4621)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4282)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4212)
    at java.awt.Container.dispatchEventImpl(Container.java:2129)
    at java.awt.Window.dispatchEventImpl(Window.java:2478)
    at java.awt.Component.dispatchEvent(Component.java:4565)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:679)
    at java.awt.EventQueue.access$000(EventQueue.java:85)
    at java.awt.EventQueue$1.run(EventQueue.java:638)
    at java.awt.EventQueue$1.run(EventQueue.java:636)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
    at java.awt.EventQueue$2.run(EventQueue.java:652)
    at java.awt.EventQueue$2.run(EventQueue.java:650)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:649)
    at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:686)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:530)
    at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:416)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:374)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)


What version of the product are you using? On what operating system?
PyCharm 2.0 Beta


```

Original issue reported on code.google.com by `Jonat...@webdancetv.com` on 19 Nov 2011 at 11:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `corbinrs...@gmail.com` on 20 Nov 2011 at 8:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 20 Nov 2011 at 5:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.10 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 23 Nov 2011 at 3:51
- Changed state: **Fixed**
- Added labels: **IntelliJ**
